docs/guide.md

```diff
@@ -0,0 +1,761 @@
+# Guide to Running E3SM
+
+## About This Guide
+
+This guide is intended to walk users step-by-step through the simulation process -- from setting up a simulation to post-processing, long-term archiving, and publishing it.
+
+## Useful Aliases
+
+Setting some aliases may be useful in running the model. You can edit your bash settings file to add aliases. Run `source` on that file to start using your new aliases. Examples:
+
+- Chrysalis – `source ~/.bashrc`
+- Compy – `source ~/.bash_profile`
+- Perlmutter – `source ~/.bash_profile.ext`
+
```

Comment:
It would be good to keep all machine-specific stuff in one file/block, so it can be updated easier. It will quickly become out-of-date in a monolithic user guide.